website/docs/misc/resources.md

```diff
@@ -4,10 +4,17 @@
 
 - [How to Localize JavaScript and React Apps with LinguiJS](https://crowdin.com/blog/2022/12/13/lingui-i18n?utm_source=lingui.dev&utm_medium=referral&utm_campaign=lingui.dev) - by [Crowdin](https://crowdin.com/?utm_source=lingui.dev&utm_medium=referral&utm_campaign=lingui.dev)
 - [How to build a fully internationalized Nextjs application using Lingui v4](https://bravo-kernel.com/blog/2023/05/how-to-build-a-fully-internationalized-nextjs-application-using-lingui-v4) by [Bravo_Kernel](https://twitter.com/bravo_kernel)
+- [The complete guide to internationalization in Next.js](https://blog.logrocket.com/complete-guide-internationalization-nextjs/) - by [Ivan Vlatkovic](https://blog.logrocket.com/author/ivanvlatkovic/)
+- [Internationalization of Next.js 13 app with App Router](https://medium.com/@amir.latypov/internationalization-of-nextjs-13-app-with-app-router-946ff017962b) - by [Amir Latypov](https://medium.com/@amir.latypov)
+- [How We Migrated a Large React Application from react-i18next to LinguiJS in 1 Day](https://medium.com/@radist2s/one-command-one-day-multiple-languages-our-migration-from-react-i18next-to-linguijs-4b07ac73a9bb) - by [Alex Batalov](https://medium.com/@radist2s)
 - [Javascript i18n with Lingui](https://mikewilliamson.wordpress.com/2017/11/05/javascript-i18n-with-lingui/) - by [Mike Williamson](https://mikewilliamson.wordpress.com/)
 
 ## Talks
 
 - [Let React speak your language](https://www.youtube.com/watch?v=soAEB7ltQPk) - by Tomáš Ehrlich
 - [Internationalization is a piece of cake](https://www.youtube.com/watch?v=vhUiL_wUAjo) - by Eli Schutze
 - [Localizing React Native apps talk from React Native EU 2022](https://www.youtube.com/live/uLicTDG5hSs?feature=share&t=7512) - by [Vojtech Novak](https://twitter.com/vonovak); covers RN but general i18n topics as well
+
+---
+
+Would like to add something to the list? Visit the [Contributing](http://localhost:3000/misc/community#contributing) section.
```

Comment:
@andrii-bodnar is localhost:3000 expected here?

Reply:
@thekip my fault, updated it in the `main` already